tv_apps/smart-system/test/marionette/app_modal_dialog_test.js

```diff
@@ -57,19 +54,16 @@ marionette.skip('Test Modal Dialog Events', function() {
   };
 
   setup(function() {
-    actions = client.loader.getActions();
     system = client.loader.getAppClass('smart-system', 'system', 'tv_apps');
+    system.waitForFullyLoaded();
     appModalDialog = new AppModalDialog(client);
-  });
-
-  function launchModalDialogApp() {
     // Launch test app
+    client.switchToFrame();
```

Comment:
I think we don't need to switch to System frame, we still stay in System frame at this line.